### PR TITLE
added listing for Nimble Storage's volume plugin

### PR DESCRIPTION
**Pull request by khudgins**
**- What I did**

Added listing for Nimble Storage's volume plugin

**- Description for the changelog**

Added listing for Nimble Storage's volume plugin

### CODE REVIEW

File: docs/extend/legacy_plugins.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ Plugin
...
 [NetApp Plugin](https://github.com/NetApp/netappdvp) (nDVP)                         | A volume plugin that provides direct integration with the Docker ecosystem for the NetApp storage portfolio. The nDVP package supports the provisioning and management of storage resources from the storage platform to Docker hosts, with a robust framework for adding additional platforms in the future.
 [Netshare plugin](https://github.com/ContainX/docker-volume-netshare)                 | A volume plugin that provides volume management for NFS 3/4, AWS EFS and CIFS file systems.
+[Nimble Storage Volume Plugin](https://infosight.nimblestorage.com/InfoSight/app#software/Integration+Kits/Linux+Toolkit+%28NLT%29)| A volume plug-in that integrates with Nimble Storage Unified Flash Fabric arrays. The plug-in abstracts array volume capabilities to the Docker administrator to allow self-provisioning of secure multi-tenant volumes and clones.
```
Comment (LK4D4) [inline docs/extend/legacy_plugins.md]: Link is not available without registration :/

Comment (khudgins) [inline docs/extend/legacy_plugins.md]: Correct. All Nimble customers will have an account on their site. I'd prefer an open link as well. If I get one, we'll update the link.

**Comment by LK4D4** [comment]
ping @thaJeztah

File: docs/extend/legacy_plugins.md (hunk 1 above)
Comment (thaJeztah) [inline docs/extend/legacy_plugins.md]: Hm, yes, I'd actually would like to have a public link as well; are you working for Nimble, or do you have contacts there?

Comment (khudgins) [inline docs/extend/legacy_plugins.md]: I'm BD at Docker. Nimble is a partner I manage, I added this entry on their behalf. I'm meeting with their staff today, let me see what I can do.

Comment (thaJeztah) [inline docs/extend/legacy_plugins.md]: Oh, lol, didn't check your name; let us know how it went :blush:

Comment (khudgins) [inline docs/extend/legacy_plugins.md]: Per Nimble, they're working on a landing page with docs and info about the plugin. From there, they will need have a link to their login-based repo for downloads, but it the landing page will be much more user-friendly with info about the plugin. Will that be acceptable?

Comment (thaJeztah) [inline docs/extend/legacy_plugins.md]: Yes, that sounds good; just so that users can see what that plugin is all about

Comment (khudgins) [inline docs/extend/legacy_plugins.md]: Sounds good. I've let them know. I'll commit a new link and update the conversation once they give it to me.

**Comment by vdemeester** [comment]
@khudgins let's close this for now. Please ping me when you get the link, I'll re-open the PR and you'll be able to update :angel:

**Comment by khudgins** [comment]
Ping @vdemeester 

Nimble has provided a landing page, I can resolve this now. Can you re-open the PR?

**Comment by thaJeztah** [comment]
@khudgins thanks! Reopening so that you can update the PR

**Comment by khudgins** [comment]
I've updated the URL. Apparently, there's something in my merge which isn't signed - all my commits should be.

Can you cherry-pick the critical commit daf4750b568169ca3895508549f5b19bf3da9647 or do I need to dig around to figure out what's not happy?

**Comment by khudgins** [comment]
Should be good now, thanks for the pointer.

**Review by thaJeztah — APPROVED** [review]
perfect! LGTM

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**